docs/release_notes/august_2022.md

```diff
@@ -0,0 +1,61 @@
+# August 2022
+## Airbyte  [v0.39.27-alpha](https://github.com/airbytehq/airbyte/releases/tag/v0.39.27-alpha) to [v0.39.41.alpha](https://github.com/airbytehq/airbyte/releases/tag/v0.39.41-alpha) 
+
+This page includes new features and improvements to the Airbyte Cloud and OSS platforms. 
+
+### New features
+* Added per-stream state to the Airbyte Cloud and OSS platforms. Per-stream state currently includes per-stream resets and connection states, and it lays the groundwork for auto-detecting schema changes, parallel syncs, and more.  
+
+    * The [new flow](https://docs.airbyte.com/cloud/managing-airbyte-cloud#edit-stream-configuration) gives you the option to refresh streams when saving changes to a connection. [#14634](https://github.com/airbytehq/airbyte/pull/14634)  
+
+    * Per-stream reset functionality is now available for connections with a Postgres source. Per-stream resets allow you to reset only the affected streams when saving an edited connection, instead of resetting all streams in a connection. [#14634](https://github.com/airbytehq/airbyte/pull/14634)  
+
+    * For connections with a Postgres source, the state of the connection to the source is displayed in the Connection State. [#15020](https://github.com/airbytehq/airbyte/pull/15020)  
+
+    * For Airbyte Open Source users:
+        * If you are using the [Postgres](https://docs.airbyte.com/integrations/sources/postgres) source connector, upgrade your Airbyte platform to version v0.40.0-alpha or newer and [upgrade](https://docs.airbyte.com/operator-guides/upgrading-airbyte/) your AzureBlobStorage connector to version 0.1.6 or newer. [#15008](https://github.com/airbytehq/airbyte/pull/15008)  
+
+* Added ``airbyte_type`` to normalization. This displays whether ``timestamp`` and ``time`` have an associated time zone.  [#13591](https://github.com/airbytehq/airbyte/pull/13591)  
```

Comment:
not sure if the double-ticks (``) will format the text correctly. Can you convert them into single ticks throughout the doc?